contracts/cw3-flex-multisig/src/state.rs

```diff
@@ -1,16 +1,29 @@
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
+use cosmwasm_std::Addr;
 use cw4::Cw4Contract;
 use cw_storage_plus::Item;
 use cw_utils::{Duration, Threshold};
 
+/// Defines who is able to execute proposals once passed
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub enum Executor {
```

Comment:
I went to the issue to add a comment, then saw you had implemented it (thank you!)

My idea was to only use `Option<Executor>` in the InstantiateMsg and store `Executor` in Config.
This also means to add one more enum variant: `Everyone`.

~~When instantiating, if the message has `None`, we assign the default `Executor::Everyone`, but then the application of the default / meaning of "None" is made on init and not later~~ (see below)